lib/setting/settings_screen.dart

```diff
@@ -1,8 +1,50 @@
+import 'dart:ffi';
+
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class SettingsWidget extends StatefulWidget{
```

Comment:
```suggestion
class SettingScreen extends StatefulWidget{
```
Caso optou seguir o padrão.